### PR TITLE
hotfix for view pagers

### LINKED ISSUE #3281: Articles pager is possibly broken in V3
# Problem
The articles pager seems to be broken in V3. 

# Steps to reproduce
Go to https://sandbox.prod.drupal.uiowa.edu/test-articles-pager-list and try to use the pager. 

# Proposed solution
?

### PR DESCRIPTION
**Pull request by joewhitsitt**
Resolves https://github.com/uiowa/uiowa/issues/3281

The way the articles and people view block pagers were built conflicts with the recent ListBlock which sets usesExposed to false even though items to display is being set. I'd like to talk about a long term solution later, but for now, this adds an exception for the know views blocks built this way.

## To Test

- [v3 people](http://sandbox.local.drupal.uiowa.edu/people-directory)/[articles](http://sandbox.local.drupal.uiowa.edu/articles) blocks should page correctly and no filters should display to the user.
- [Student Profiles Grid](http://admissions.local.drupal.uiowa.edu/student-profiles/aasthika-das) on Admissions should still filter on profiles (layout builder page or student profiles template) but no filters should display to the user.

### CODE REVIEW

File: docroot/modules/custom/layout_builder_custom/src/Plugin/Display/ListBlock.php [hunk 1]
```diff
@@ -246,6 +246,17 @@ public function usesExposed() {
...
     }
+    // Hotfix shim to keep these pagers working for now.
+    // @todo Solve ListBlock paging.
+    $display = $this->view->getDisplay();
+    $exceptions = [
+      'block_people_slf',
+      'block_people_sfl',
+      'block_articles'
+    ];
+    if (in_array($display->display["id"], $exceptions)) {
```
Comment (pyrello) [inline docroot/modules/custom/layout_builder_custom/src/Plugin/Display/ListBlock.php]: Can we switch this to single quote?

**Comment by pyrello** [comment]
Pulling down for testing.

File: docroot/modules/custom/layout_builder_custom/src/Plugin/Display/ListBlock.php (hunk 1 above)
Comment (joewhitsitt) [reply docroot/modules/custom/layout_builder_custom/src/Plugin/Display/ListBlock.php]: glad to have you back!

**Review by pyrello — APPROVED** [review]
Tested and this looks good. Thanks!

**Pull request merged**